packages/design-system/src/icons/svgs/third-party-brands/brand-medium-24.icon.ts

```diff
@@ -0,0 +1,26 @@
+import {html} from 'element-vir';
+import {defineIcon} from 'vira';
+import {toniqIconCssVarWithFallback} from '../../../styles';
+
+export const BrandMedium24Icon = defineIcon({
+    name: 'BrandMedium24Icon',
+    svgTemplate: html`
+        <svg
```

Comment:
This Medium logo is out of date:
![medium](https://github.com/Toniq-Labs/toniq-labs-design-system/assets/1205860/a38f2058-a197-4284-9350-3aa7cbc43fbc)

And here's the SVG for it:
```

        <svg
            xmlns="http://www.w3.org/2000/svg"
            stroke="none"
            fill=${toniqIconCssVarWithFallback['toniq-icon-fill-color']('#000')}
            width="24"
            height="24"
            viewBox="0 0 24 24"
        >
            <path
                d="M13.4 12a6.23 6.23 0 0 1-6.2 6.25C3.79 18.25 1 15.45 1 12a6.23 6.23 0 0 1 6.2-6.25c3.43 0 6.2 2.8 6.2 6.25m6.82 0c0 3.25-1.4 5.88-3.1 5.88-1.72 0-3.1-2.63-3.1-5.88s1.38-5.88 3.1-5.88c1.7 0 3.1 2.63 3.1 5.88M23 12c0 2.91-.49 5.27-1.1 5.27-.6 0-1.08-2.36-1.08-5.27 0-2.9.49-5.27 1.09-5.27C22.5 6.73 23 9.1 23 12"
                style="fill-rule:nonzero"
            />
        </svg>
```